lib/puppet/module_tool/metadata.rb

```diff
@@ -174,24 +174,24 @@ def validate_name(name)
 
       err = case modname
       when nil, '', :namespace_missing
-        "the field must be a namespaced module name"
+        _("the field must be a namespaced module name")
       when /[^a-z0-9_]/i
-        "the module name contains non-alphanumeric (or underscore) characters"
+        _("the module name contains non-alphanumeric (or underscore) characters")
```

Comment:
Wow these regexes will prevent us from having any module names with UTF-8 characters. I wonder if there's a bug for that already. cc @Iristyle